python/mxnet/runtime.py

```diff
@@ -27,17 +27,27 @@
 
 class Feature(ctypes.Structure):
     """
-    Compile time feature description
+    Compile time feature description, member fields: `name` and `enabled`.
     """
     _fields_ = [
         ("_name", ctypes.c_char_p),
-        ("enabled", ctypes.c_bool)
+        ("_enabled", ctypes.c_bool)
     ]
 
     @property
     def name(self):
+        """
+        feature name
```

Comment:
```suggestion
        Feature name.
```
Can the name be anything? Spaces, special characters? Might be a good spot to provide more direction.